monitoring/README.md

```diff
@@ -4,43 +4,83 @@
+prometheus-kube-prometheus-operator-79d5cf844f-nbzvb   1/1     Running   0          2m57s
+prometheus-prometheus-kube-prometheus-prometheus-0     3/3     Running   0          2m54s
+pyroscope-0                                            1/1     Running   0          92s
+pyroscope-alloy-0                                      2/2     Running   0          92s
+thanos-compactor-c8756458-knh5r                        1/1     Running   0          2m26s
+thanos-query-c796b4647-mrjnb                           1/1     Running   0          2m26s
+thanos-query-frontend-5fb66b5744-rdklf                 1/1     Running   0          2m26s
+thanos-storegateway-0                                  1/1     Running   0          2m26s
+```
+
+
+b) Deploy the monitoring tools in a WEC cluster (e.g., `cluster1`):
+
 ```bash
-$ cd monitoring
-$ ./install-ks-monitoring.sh
+./install-ks-monitoring.sh --cluster-context cluster1 --set wec
 ```
 
 Optionally, check the deployment of the monitoring tools:
 
 ```bash
-$ kubectl -n ks-monitoring get pods
-NAME                                                  READY   STATUS    RESTARTS   AGE
-grafana-7969f6cf5c-lp4g6                              1/1     Running   0          66s
-prometheus-kube-prometheus-operator-b56d747c9-wfs2w   1/1     Running   0          42s
-prometheus-prometheus-kube-prometheus-prometheus-0    2/2     Running   0          39s
-pyroscope-0                                           1/1     Running   0          90s
-pyroscope-alloy-0                                     2/2     Running   0          90s
+kubectl -n ks-monitoring get pods
+```
+
+Output:
+```bash
+NAME                                                   READY   STATUS    RESTARTS   AGE
+prometheus-kube-prometheus-operator-79d5cf844f-nbzvb   1/1     Running   0          2m57s
+prometheus-prometheus-kube-prometheus-prometheus-0     3/3     Running   0          2m54s
+pyroscope-0                                            1/1     Running   0          92s
+pyroscope-alloy-0                                      2/2     Running   0          92s
 ```
 
-#### 2. Configure Prometheus and Pyroscope to scrape your KubeStellar spaces:
+
+
+#### 2. Configure Prometheus and Pyroscope to scrape your KubeStellar spaces and Workload Execution Clusters:
 
 a) WDS space:
 
 ```bash
-$ ./configure-metrics-wds.sh
+./configure-monitoring-wds.sh
 ```
 
-If you have more than one WDS, you can run the above script with the `--space-name` flag. For example: `./configure-metrics-wds.sh --space-name wds2`
+If you have more than one WDS, you can run the above script with the `--space-name` flag. For example: `./configure-monitoring-wds.sh --space-name wds2`
 
 Optionally, check the prometheus service monitor objects for the WDS space:
 
 ```bash
-$ kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack" | grep wds1*
+kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack" | grep wds1*
```

Comment:
This fails due to the lack of quoting around `wds1*`.

Reply:
Ok, I see! Fixed but it works for me even without any quotes:

```
$ kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack" | grep wds1*
wds1-apiserver                 3d19h
wds1-ks-controller-manager     3d19h
wds1-ks-transport-controller   3d19h
```

```
$ kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack" | grep its1*
its1-apiserver                 3d19h
its1-status-addon-controller   3d19h
```